test/specs/accounts/update.account.spec.js

```diff
@@ -1,57 +1,40 @@
 'use strict';
-let config = require('../../../config.json');
 let loginPage = require('../../pages/login.page');
 let topSideBar = require('../../pages/topsidebar.page');
-let content = require('../../pages/accounts/acountContent.page');
-let form = require('../../pages/accounts/accountForm.page');
-let expect = require('chai').expect;
+let content = require('../../pages/content.page');
+let form = require(`../../pages/accounts/accountForm.${theme}.page`);
+
 describe('Acceptance Test to Modify an Account', function () {
-    this.retries(3);
 
-    let accountModifiedData = {
-        name: 'Account Updated',
+    let accountModified = {
+        name: 'Account Sample',
         number: 100,
         site: 2,
-        parent: 'Edge Communications',
         type: 'Other',
         industry: 'Agriculture',
         annualRevenue: 200,
         rating: 'Hot',
-        phone: 591591591,
-        fax: 591591222,
-        website: 'www.website.com',
-        tickerSymbol: 'ACC',
-        ownership: 'Private',
-        employees: 10,
-        sicCode: 200200200
     };
 
-    let accountToModify = 'Account to update';
+    let accountToModify = {
+        name: 'Account Required'
+    };
 
     beforeEach(function () {
-        // Login on application.
-        loginPage.login(config.username, config.password);
-        // Create new account to modify.
+        loginPage.login(loginApplication.username, loginApplication.password);
         topSideBar.goToSection('accounts');
         content.clickOnNewButton();
-        form.fillAccountWithRequiredFieldsAndSave(accountToModify);
-
-    });
-
-    // Delete an accounts created.
-    afterEach(function () {
```

Comment:
maybe you will need this afterEach to delete account hast been created before.

Reply:
the same as above